charts/ocp-proxy-api/templates/configmap.yaml

```diff
@@ -63,6 +63,10 @@ data:
             deny all;
             {{- end }}
 
+            proxy_connect_timeout   {{ default 10 .Values.timeouts.proxy_connect_timeout }};
```

Comment:
add these timeout defaults to the values.yaml file, vs just the empty dict.

Reply:
done